### PR TITLE
All Changes Complete

### PR DESCRIPTION
**Pull request by samiullah997**
- update the functionality of the filter section of the project.
- update my readme file
- update the live video link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Check the live demo here [Live Demo](https://github.com/samiullah997/thired-caps
 
 ### LIVE VIDEO LINK
 
-Check the live demo here [Video Links](https://www.loom.com/share/59121c98742a40a997f3875811a51618)
+Check the live demo here [Video Links](https://www.loom.com/share/d138d8d6ca024989ae684e59c1b8680c)
```
Comment (alexander16108) [inline README.md:48]: - as required by microverse and the previous code review, you should have a live demo link to your project
I suggest you deploy your web application using `netlify`  as it is the most wildly used to deploy applications.  😄 
cheers and good luck 👍🏾

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @samiullah997 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
don't give up ur almost there just a few more steps to go and your project will be ready 👍🏾 



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by alexander16108** [comment]
- @samiullah997  please do not create a new pull request to add you changes, deploy your app and submit your project

**Review by DammyShittu — APPROVED** [review]
Hi @samiullah997,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**